code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -126,10 +126,11 @@
 		return src.close()
 	return src.open()
 
-// this should probably use dump_contents()
 /obj/structure/closet/ex_act(severity, target)
 	contents_explosion(severity, target)

```

Comment:
I really don't get it... the `open()` two lines below was already calling dump_contents()...